deegree-services/deegree-services-config/pom.xml

```diff
@@ -17,7 +17,7 @@
 
   <dependencies>
     <dependency>
-      <groupId>org.deegree</groupId>
+      <groupId>${project.groupId}</groupId>
```

Comment:
Please align this to the deegree codebase and use the groupId without variable.

Reply:
Solved in PR #938.